src/replication.c

```diff
@@ -764,6 +1006,35 @@ get_non_quorum_replica_count(spec_t *spec)
 	return replica_count;
 }
 
+/*
+ * This will return true if given replica need to be updated to vol_mgmt
+ * as trusty one during replica connect phase. This should be called by
+ * holding spec->rq_mtx lock
+ */
+static bool
+needs_trusty_replica_update_during_connect(spec_t *spec, replica_t *replica) {
+	int trusty_replica_count = 0;
+
+	ASSERT(MTX_LOCKED(&spec->rq_mtx));
+	/*
+	 * Update trusty replica if its count < RF and quorum on
+	 * and ID also doesn't exist in list yet
+	 * i.e., new volume or upgrade from 1.2 to 1.3
+	 */
+	trusty_replica_count = get_trusty_replica_count(spec);
+ 	if ((is_trusted_replicas_contain_replicaid(spec, replica->replica_id) == false) &&
```

Comment:
Can we change it to something like `is_replica_trusted`? Through `replica_id` we decide if a replica is trusted or not, so we can drop `replicaid` from function name.

Reply:
We are already having a function `check_is_it_trusty_replica`. So `is_replica_trusted` is also the same which will be conflicting.